embargo-policy.md

```diff
@@ -0,0 +1,30 @@
+# Embargo Policy
+
+This policy forbids members of this project's security contacts <!-- TODO: $LINK --> and others
+defined below from sharing information outside of the security contacts and this
+listing without need-to-know and advance notice.
+
+The information members and others receive from the list defined below must:
+
+* not be made public,
+* not be shared,
+* not be hinted at
+* must be kept confidential and close held
+
+Except with the list's explicit approval. This holds true until the public
+disclosure date/time that was agreed upon by the list.
+
+If information is inadvertently shared beyond what is allowed by this policy,
+you are REQUIRED to inform the [security contacts](https://github.com/konveyor/security/SECURITY_CONTACTS.md) of exactly what
+information leaked and to whom. A retrospective will take place after the leak
+so we can assess how to not make this mistake in the future.
+
+Violation of this policy will result in the immediate removal and subsequent
+replacement of you from this list or the Security Contacts.
+
+## Disclosure Timeline
+
+This project sustains a **60 days disclosure timeline** to ensure we provide a
```

Comment:
@jwmatthews : do you think 60 days disclosure timeline is acceptable?

Reply:
If I understand, this means that within 60 days of being informed we will aim to address the issue and release a fix, along with making the issue public?  That sounds reasonable to me.

Reply:
Yes. if we need more time to fix, we have a provision in the policy for communication and extend the timeline.